### DISCUSSION TITLE
Cant access firefly anmore

### DISCUSSION THREAD

**Discussion by Vergissmeinnicht12** [iotaledger/firefly#7692] [Help & Support]
I have a stronghold backup, but it says the password is wrong. But it must be the right one, I have written it down.

Why cant I login anymore with Email + PW in the first place?  Firefly is always asking for a backup now

**Comment by begonaalvarezd** [comment]
If it says the password is wrong, unfortunately it must be wrong, maybe you changed it at some point. You can restore your profile using your 24 words menmonic if you dont remember your stronghold password
> Why cant I login anymore with Email + PW in the first place
Im not sure I understand what you mean here, it was never possible to login using email&password

**Comment by Vergissmeinnicht12** [comment]
Please can you respond: Why cant I login anymore with Email + PW in the first place? Firefly is always asking for a backup now - why? I was logging in very often since the chrysalis update.

I dont have 24 words, I only have the stronghold backup + the credentials that were always working. How can I access the funds now?

**Reply by begonaalvarezd** [reply]
Are you sure you used Firefly before? We have never had email + password to enter a profile, the only things we have are:
- Profile name
- Profile pin (6 digits to enter the profile)
- Profile password (not for ledger profiles, used to encrypt stronghold)
- 24 words mnemonic (this is your seed and you should try to never lose this)

If you dont have your 24 words nor the password to your stronghold or a ledger profile, Im afraid I wont be able to help you, as you should be in control of your keys

**Reply by Vergissmeinnicht12** [reply]
Yes I meant profile name, where can I access that?

**Reply by begonaalvarezd** [reply]
If you dont have an existing profile in Firefly already, you need to import one using your stronghold plus the correct password to decyrpt it, your 24 words, or a ledger device

**Reply by Vergissmeinnicht12** [reply]
I do have an existing profile - how can I access it with Profile + PW as I did many times before? Why is Firefly always asking for a backup now?

Is the password for the stronghold backup different from the one used to log in with the profile?

**Reply by begonaalvarezd** [reply]
What do you mean with firefly always asking for a backup?

**Reply by begonaalvarezd** [reply]
If you already have a profile and are trying to access your profile after the network upgrade (please make sure you are using firefly version 2.0.3), you will potentially be asked for 2 values
- pin: 6 numbers
- stronghold password: to restore your profile after the network upgrade, this will not always be requested, just if your profile needs it. Your stringhold password will also be asked if you are running an old version of stronghold, and firefly required an internal stronghold upgrade for which it also needs the password

**Reply by Vergissmeinnicht12** [reply]
I always get this screen when opening Firefly: https://ibb.co/JdZSVXP

Why cant i just login as before?

**Reply by begonaalvarezd** [reply]
Are you using the latest version of firefly? Please make sure you are running the latest version and that you do not have any other version installed in parallel which can lead to that issue

**Comment by Vergissmeinnicht12** [comment]
ok I was using an older version and that was the issue. I would suggest that Firefly mentions that there is a newer version available and that it is mandatory.

Do I have to manually unlock the ASMB funds in order to get IOTA for it? I never did that, does it mean I missed some tokens?

**Reply by begonaalvarezd** [reply]
> Do I have to manually unlock the ASMB funds in order to get IOTA for it

No need to collect, the outputs are yours
Will close the discussion as the issue seems fixed 🌷

**Comment by Vergissmeinnicht12** [comment]
It is so annoying that one is forced to continue holding IOTA and can only gradually dump it, even if one considers IOTA a bad joke after the supply increase. I feel it's humiliating for me to support such a lousy project with my funds. Bu this is the evolution of Satoshis vision, am I right?
